### PR TITLE
[11.0] [MIG] stock_scanner_location_info

### PR DESCRIPTION
**Pull request by misern2**
Standard Migration

Depends on #149

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please rebase

**Comment by rousseldenis** [comment]
@misern2 Please attend @pedrobaeza comments

**Comment by misern2** [comment]
@rousseldenis Rebase done.

**Review by rousseldenis — APPROVED** [review]

File: stock_scanner_location_info/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# © 2011-2015 Sylvain Garancher <sylvain.garancher@syleam.fr>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Stock Scanner Location Info',
+    'version': '11.0.1.0.0',
+    'category': 'Generic Modules/Inventory Control',
+    'website': 'https://odoo-community.org/',
```
Comment (tbaden) [inline stock_scanner_location_info/__manifest__.py]: please change website to: https://github.com/OCA/stock-logistics-barcode

**Review by tbaden — CHANGES_REQUESTED** [review]

**Comment by rousseldenis** [comment]
@tbaden

**Review by tbaden — APPROVED** [review]

**Pull request merged**